File: swagger/models/get_class_visits_response.py
from .base_model import BaseModel

from .class import Class

class GetClassVisitsResponse(BaseModel):
	"""
	no description available

	Attributes:

		class
		Contains class and booking information.
	"""

	input_map = {
		'Class': ('class', Class),
		}

	output_map = {
		'class': 'Class',
		}

	class = None

File: swagger/Models/GetClassVisitsResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetClassVisitsResponse extends BaseModel
{
	/**
	 * Contains class and booking information.
	 * 
	 * @var Class
	 */
	public $class;

	protected function getInputMap()
	{
		return [
			'Class' => ['class', Class::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'class' => 'Class',
			];
	}
}
